server/src/test/java/org/elasticsearch/discovery/SnapshotDisruptionIT.java

```diff
@@ -49,7 +48,6 @@
 /**
  * Tests snapshot operations during disruptions.
  */
-@ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 0, transportClientRatio = 0, autoMinMasterNodes = false)
```

Comment:
I think this shouldn't be removed, although the `autoMinMasterNodes = false` can go.